src/Cosmos/ProtobufSerialization.cpp

```diff
@@ -212,6 +212,20 @@ google::protobuf::Any convertMessage(const Proto::Message& msg) {
                 return any;
             }
 
+        case Proto::Message::kMayachainSendMessage:
```

Comment:
What do you think if we'll reuse `ThorchainSendMessage` for MAYAChain? I don't see the reasons why we need to have `MayachainSendMessage`

Reply:
Done, already remove that one